### PR TITLE
1129 - Consistency Checker (#33 assigned)

### PR DESCRIPTION
**Pull request by sayeed1999**
Editorial on 1129 - Consistency Checker problem on lightoj.
It is a Trie based problem. Check!

### CODE REVIEW

File: 1129/en.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# 1129 - Consistency Checker
```
Comment (Tahanima) [inline 1129/en.md:1]: Please follow the format we use for the problem heading - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR1

File: 1129/en.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+**Problem Link:** [click here](http://lightoj.com/volume_showproblem.php?problem=1129)
```
Comment (Tahanima) [inline 1129/en.md:3]: Please remove the problem link.

File: 1129/en.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+**Algorithm:** [Trie Tree Data Structure on Shafaetsplanet](http://www.shafaetsplanet.com/?p=1679) **Shafaetsplanet** blog has really awesome tutorial on trie data structure. Also you can learn from anywhere.
+***Some youtube resources on trie:-***
+- [Tushar Roy - Coding Made Simple](https://m.youtube.com/watch?v=AXjmTQ8LEoI)
+- [HackerRank](https://m.youtube.com/watch?v=zIjfhVPRZCg)
+
```
Comment (Tahanima) [inline 1129/en.md:9]: Either list down the resources/other tutorials to solve the problem in the end or try to combine them in your writing. I would prefer the latter. Here's a reference - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR7

File: 1129/en.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+### Theory:
+If you have learnt **trie data structure** well, this problem is really easy for you.
+
+You will be given some numbers with length 1-10. And you have to find **"is atleast one number a prefix of another number or not"**. (To recap, a substring from the 0th index of a string is a prefix of that string. Ex- **123** is a prefix of **12345**, but **124**, **132** or **234** is not a prefix of **12345**.) So simply take the numbers as strings and insert the strings into the trie tree.
+
+**How will be the trie node?!** Since a number can have digit only between 0-9, then the trie node will have one boolean variable and an array of trie nodes of length 10. Once you complete inserting the numbers as strings into the trie, you will write a function which will check if a single string is a prefix of another or not, and the function will return true or false. Based on that, you got your answer. :)
+
+### Code: (C++ solution)
```
Comment (Tahanima) [inline 1129/en.md:17]: Keep the format consistent with this one - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR19

File: 1129/en.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+This is an accepted solution of mine!
```
Comment (Tahanima) [inline 1129/en.md:18]: Please remove this line - `This is an accepted solution of mine!`

**Review by Tahanima — CHANGES_REQUESTED** [review]
Hi,
Please resolve the requested changes.

**Review by sayeed1999 — COMMENTED** [review]
Modified. Thanks

**Pull request merged**